webdriver/webdriver.go

```diff
@@ -30,15 +30,41 @@ func GetWebDriver() (*selenium.Service, selenium.WebDriver, error) {
 	panic("Invalid --browser value specified")
 }
 
-func FindShadowElements(d selenium.WebDriver, e selenium.WebElement, selector string) ([]selenium.WebElement, error) {
-	i := []interface{}{e}
-	result, err := d.ExecuteScriptRaw(fmt.Sprintf("return arguments[0].shadowRoot.querySelectorAll('%s')", selector), i)
-	if err != nil {
-		panic(err.Error())
+func FindShadowElements(
```

Comment:
Note: the changes to this function allow you to repeatedly resolve shadow dom for multiple elements. 

If `foo` has shadow `bar` which has shadow `baz`, we can now call

    FindShadowElements(wd, foo, "bar", "baz")

instead of having to handle that ourselves at the call site.